packages/next/src/server/config.ts

```diff
@@ -262,6 +262,23 @@ function assignDefaults(
     )
   }
 
+  // TODO: remove once we've made PPR default
+  // If we're testing, and the `__NEXT_EXPERIMENTAL_PPR` environment variable
+  // has been set to `true`, enable the experimental PPR feature so long as it
+  // wasn't explicitly disabled in the config.
+  if (
+    process.env.__NEXT_TEST_MODE &&
+    process.env.____NEXT_EXPERIMENTAL_PPR === 'true' &&
+    result.experimental?.ppr === undefined
+  ) {
+    result.experimental ??= {}
+    result.experimental.ppr = true
```

Comment:
I wonder if doing this in our `createNext` setup here would be a bit cleaner/more reliable for changing next config values: https://github.com/vercel/next.js/blob/canary/test/lib/e2e-utils.ts#L155-L179

Reply:
Indeed, probably better to add it to `createNext` if possible.

Reply:
For tests that define a configuration via the `createNextDescribe` interface yes, this is a better path. This doesn't however help when the test defines a `next.config.js`. The solution I've implemented here handles all code paths as all code paths load the initial configuration via the `loadConfig` function. Ideally, we'd have a separate `coerce` function that could apply customizations like this, but this ensure that every code path that gets the configuration will hit this case.